net-misc/asterisk/asterisk-18.24.1.ebuild

```diff
@@ -105,18 +101,21 @@ BDEPEND="dev-libs/libxml2:2
 
 QA_DT_NEEDED="/usr/lib.*/libasteriskssl[.]so[.][0-9]\+"
 
-_make_args=(
-	"NOISY_BUILD=yes"
-	"ASTDBDIR=\$(ASTDATADIR)/astdb"
-	"ASTVARRUNDIR=/run/asterisk"
-	"ASTCACHEDIR=/var/cache/asterisk"
-	"OPTIMIZE="
-	"DEBUG="
-	"DESTDIR=${D}"
-	"CONFIG_SRC=configs/samples"
-	"CONFIG_EXTEN=.sample"
-	"AST_FORTIFY_SOURCE="
-)
+_ast_make() {
```

Comment:
For the record, you don't need to `_` your functions, the namespace is yours.

Reply:
Someone previously said I should, would be extremely happy to drop that.  Would you prefer I do that prior to merge?

Reply:
Done.